src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+---
+title: Active areas of Ethereum research
+description: Explore different areas of open research and learn how to get involved.
+sidebar: true
+lang: en
+---
+
+# Open areas of Ethereum research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
+
+Regardless of the specific topic, there is a wealth of information on Ethereum research to be found at [ethresear.ch](https://ethresear.ch) and the [Eth R&D Discord channel](https://discord.gg/n7JxAeRu). This report from [DelphiDigital](https://members.delphidigital.io/reports/the-hitchhikers-guide-to-ethereum) also covers a lot of ground. These are the primary places where Ethereum researchers discuss the latest ideas and development opportunities.
+
+## Scaling and performance
+
+There are ongoing efforts to scale Ethereum using a variety of techniques including sharding the blockchain, danksharding and using rollups. Introductory information on scaling Ethereum is available on our [scaling page](/developers/docs/scaling).
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+#### Background reading
+
+[Introduction to layer 2](/layer-2/)
+
+[EthHub Layer-2](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)
+
+[Polynya: Rollups, DA and modular chains](https://polynya.medium.com/rollups-data-availability-layers-modular-blockchains-introductory-meta-post-5a1e7a60119d)
+
+#### Recent research
+
+[Arbitrum's fair-ordering for sequencers](https://eprint.iacr.org/2020/269.pdf)
+
+[ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+[Rollup-centric roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698)
+
+
+### Bridges
+
+One particular area of layer 2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2. This is a particularly important area of researchg because bridges are commonly targeted by hackers.
```

Comment:
```suggestion
One particular area of layer 2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2. This is a particularly important area of research because bridges are commonly targeted by hackers.
```